examples/shared/dtlsconnection.c

```diff
@@ -554,7 +554,7 @@ void connection_free(dtls_connection_t * connList)
 int connection_send(dtls_connection_t *connP, uint8_t * buffer, size_t length){
     if (connP->dtlsSession == NULL) {
         // no security
-        if ( 0 != send_data(connP, buffer, length)) {
+        if (0 > send_data(connP, buffer, length)) {
```

Comment:
This would treat 0 length as an error. I would suggest "0 >=" instead. Of course send_data doesn't pass a 0 length on to sendto, but that is a different issue.

Reply:
@tobbe303, what do you think ? do you plan to adapt the code if you agree ?

@sbertin-telular, if we don't get any news in few days I will integrate your fix. (I mean using `>=`)

Reply:
@sbernard31 Agree. I've updated the code as suggested.